inc/scroom/layeroperations.hh

```diff
@@ -69,11 +106,11 @@ public:
                     Scroom::Utils::Stuff cache);
 };
 
-class Operations24bpp : public CommonOperations
+class Operations24bpp : public PipetteCommonOperationsRGB
 {
 public:
-  static Ptr create();
-  Operations24bpp();
+  static PipetteCommonOperations::Ptr create(int bps);
```

Comment:
For RGB 24bpp, bps is always 8, right? No need to pass it in

Reply:
... I see that something similar holds for the other concrete classes below.

Reply:
Do you mean something like:
```cpp
PipetteCommonOperationsCMYK::Ptr OperationsCMYK32::create()
{
  return PipetteCommonOperationsCMYK::Ptr(new OperationsCMYK32());
}

OperationsCMYK32::OperationsCMYK32() : PipetteCommonOperationsCMYK(8)
{
}
```
in e.g. `cmyklayeroperations.cc` and similarly for `layeroperations.cc`?

Reply:
Yes, that would be it.